react/utils/index.tsx

```diff
@@ -151,3 +151,17 @@ export const orderSplitMessage = ({
 
   return intl.formatMessage(message, { deliveries, pickups, takeaways })
 }
+
+
+
+export function parseBankInvoiceUrl(url: string) {
+  const isEncrypted = Boolean(url.match(/(\*.\*.)+\*\w\*/g))
+  if (!isEncrypted) return url
+
+  const rootPath = window?.__RUNTIME__.rootPath ?? ''
```

Comment:
Why is this necessary? Wouldn't `useRuntime` work in this case?

Reply:
I just copied it from `order-placed` :sad: